packages/cli/src/lib/snapshot/snapshot.spec.ts

```diff
@@ -46,49 +95,101 @@ const doMockAuthenticatedClient = () => {
 describe('Snapshot', () => {
   beforeAll(() => {
     doMockAuthenticatedClient();
-    doMockReadStream();
   });
 
-  describe('when the the resources are compressed', () => {
-    const pathToZip = join('dummy', 'path');
+  describe('when the resources are in error', () => {
+    let snapshot: Snapshot;
+    let initialSnapshotState: ResourceSnapshotsModel;
+    let status: ISnapshotValidation;
+    const targetOrgId = 'target-org';
+    const snapshotId = 'target-org-snapshot-id';
+
+    const doMockGetSnapshotInError = async () => {
+      const errorReport = getErrorReport(snapshotId);
+      const futureSnapshotState = await getDummySnapshotModel(
+        targetOrgId,
+        snapshotId,
+        [errorReport]
+      );
+
+      mockedGetSnapshot.mockResolvedValueOnce(futureSnapshotState);
+    };
 
     beforeEach(async () => {
-      await SnapshotFactory.createFromZip(pathToZip, 'my-target-org');
-    });
+      doMockGetSnapshotInError();
```

Comment:
await?
```suggestion
      await doMockGetSnapshotInError();
```

Reply:
mmh non. I think the doMockGetSnapshotInError should not return a promise

Reply:
Good catch tho